plenty_api/api.py

```diff
@@ -1308,6 +1310,17 @@ def plenty_api_get_shipping_packages_for_order(self, order_id: int,
         return utils.summarize_shipment_packages(
             response=package_responses, mode=mode)
 
+    def plenty_api_get_amazon_product_types(self):
+        """
+        Get a list of all available Amazon product types.
+        https://developers.plentymarkets.com/en-gb/plentymarkets-rest-api/index.html#/Pim/get_rest_pim_amazon_product_types
+        Return:
+                        [JSON(Dict) / DataFrame] <= self.data_format
+        """
+        domain = 'pim' # pim is the REST Route not the domain
+        path = '/amazon-product-types' # path in this case is the endpoint
+        return self.__plenty_api_generic_get(domain='pim', path=path)
```

Comment:
What does this return? Do we have the chance to process the result a bit, to ease the work for the user? Because all we actually want is a mapping of the ID of a product type to the name of the product type. Thus if there isn't any other valuable data in that result, then we should just return a dictionary of these two things. If there is more valuable information then I would opt for at least a 'minimal' parameter, which allows the user to just get this dictionary.

Reply:
Here is a sample of the return value data struct.

```json
[
  {
    "id": 92,
    "genre": "AutoAccessory",
    "type": "Abbigliamento moto",
    "genreDescription": "AutoAccessory",
    "typeDescription": "Abbigliamento moto"
  },
  {
    "id": 93,
    "genre": "AutoAccessory",
    "type": "Accessories",
    "genreDescription": "AutoAccessory",
    "typeDescription": "Accessories"
  }
]
```

It is a complete list of dicts. No Parameters - or optionals.
I would not touch or transform it on SDK level, this should be done on user level - if they want to modify it they can.

Reply:
I have done this transform step before on this SDK, so it wouldn't be something abnormal. But we can keep it as is for now.